.github/workflows/repackage.yml

```
@@ -61,6 +61,16 @@ jobs:
 
           ./scripts/upload-bundles "${BUILD_DATA_DIR}" "$KERNEL_BUNDLE_BUCKET"
 
+      - name: Check BTF support
+        run: |
+          gsutil cp "$KERNEL_BUNDLE_BUCKET/kernel-features.json" .
+
+          ./scripts/kernel-features.py --output kernel-features.json "${BUILD_DATA_DIR}/bundles"
+
+          gsutil cp kernel-features.json "$KERNEL_BUNDLE_BUCKET"
```

[ultra-nit] Could we download the file to /tmp? I think as is right now it downloads into the repo directory and we do things with it (nasty things no one should do). I just don't want the file being accidentally committed to the repo.
```suggestion
          tempdir="$(mktemp -d)"
          gsutil cp "$KERNEL_BUNDLE_BUCKET/kernel-features.json" "$tempdir/"

          ./scripts/kernel-features.py --output "$tempdir/kernel-features.json" "${BUILD_DATA_DIR}/bundles"

          gsutil cp "$tempdir/kernel-features.json" "$KERNEL_BUNDLE_BUCKET"
```